configuration/product_variant/drugs.csv

```diff
@@ -416,8 +416,8 @@ init.05a16ca0-4a45-4dc5-a503-a44b61b4c283,Someticone Gouttes Oral 50mg/ml (Flaco
 init.089be2d2-1b2b-4e63-88bf-1eab528fcbb5,Sous-Salicylate de bismuth 262mg/15ml (Flacon/240),product.categ_products_drug_orders,product,init.3b10054a-3ccb-47ee-82d8-d545822ea328,init.3b10054a-3ccb-47ee-82d8-d545822ea328,TRUE,TRUE,By Lots,TRUE,Ordered quantities
 init.58f221b9-fb04-45b0-94be-0feece7db5f2,Spironolactone 25mg Comprime,product.categ_products_drug_orders,product,init.6fbbb779-94ca-4d83-8e02-8a559627b9fb,init.6fbbb779-94ca-4d83-8e02-8a559627b9fb,TRUE,FALSE,By Lots,TRUE,
 init.3a3f865b-6480-43f3-9536-4265e5a7b359,Spironolactone 25mg Comprime (PAQ/30),product.categ_products_drug_orders,product,init.4fc93bbe-bb54-4e20-ae5f-a5cec41de5fa,init.4fc93bbe-bb54-4e20-ae5f-a5cec41de5fa,FALSE,TRUE,By Lots,TRUE,Ordered quantities
-init.12d29b03-08d8-4b7d-b3d3-e15a587fe010,Spironolactone 50mg,product.categ_products_drug_orders,product,init.4801928b-f31a-4341-bdf2-c982ecb649f0,init.4801928b-f31a-4341-bdf2-c982ecb649f0,TRUE,FALSE,By Lots,TRUE,
-init.cf63a9b7-565e-4f44-9288-e120b4d3d589,Spironolactone 50mg (PAQ/30),product.categ_products_drug_orders,product,init.4fc93bbe-bb54-4e20-ae5f-a5cec41de5fa,init.4fc93bbe-bb54-4e20-ae5f-a5cec41de5fa,FALSE,TRUE,By Lots,TRUE,Ordered quantities
+init.12d29b03-08d8-4b7d-b3d3-e15a587fe010,Spironolactone 50mg Comprime,product.categ_products_drug_orders,product,init.6fbbb779-94ca-4d83-8e02-8a559627b9fb,init.6fbbb779-94ca-4d83-8e02-8a559627b9fb,TRUE,FALSE,By Lots,TRUE,

```

Comment:
@nuhuhmutebi  are you sure it is possible to change a product's UoM when some transactions already have been done?